src/ctxt.rs

```diff
@@ -892,7 +892,7 @@ impl FctKind {
 }
 
 #[derive(Copy, Clone, Debug, PartialEq, Eq)]
-pub enum Intrinsic {
+pub enum Builtin {
```

Comment:
Actually I would have preferred `Intrinsic` to `Builtin` as a name, but if you prefer `Builtin` that's fine as well.